src/components/Lookup/Lookup.js

```diff
@@ -93,6 +96,28 @@ export class Lookup extends React.Component {
      * timestamp.
      */
     allowCreate: React.PropTypes.bool,
+    /**
```

Comment:
We should be careful of adding experimental options to otherwise stable components but personally I'm ok with it. We probably have to revisit this once SF moved the Lookup-Modal to stable